### PR TITLE
 #164489878 set up empty react project

### PR DESCRIPTION
**Pull request by Jaman-dedy**
**What this PR do?**

This PR will set up an empty react project

**How should this be manually tested?**
_  this can be manually tested by running the following command `npm start`

**What are the relevant pivotal tracker stories?**
[#164489878](https://www.pivotaltracker.com/story/show/164489878)

### CODE REVIEW

**Comment by blaisebakundukize** [comment]
Good job @Jaman-dedy, I appreciate the task you did. But you have to add a new line at the end of ```.gitignore``` file. Thx

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './App.css';
+
+function App() {
```
Comment (nakiwuge) [inline src/App.js]: Please use  Es6

**Review by Paccy10 — COMMENTED** [review]
Nice job @Jaman-dedy , but I think you have to name the `allreducers.js`  file as `index.js`

File: src/reducers/allreducers.js [hunk 2]
```diff
@@ -0,0 +1 @@
+console.log('hello');
```
Comment (nakiwuge) [inline src/reducers/allreducers.js]: remove console.log

File: src/components/layouts/navbar.js [hunk 3]
```diff
@@ -0,0 +1 @@
+console.log('hello');
```
Comment (nakiwuge) [inline src/components/layouts/navbar.js]: Remove this file

File: src/actions/user.js [hunk 4]
```diff
@@ -0,0 +1 @@
+console.log('hello');
```
Comment (nakiwuge) [inline src/actions/user.js]: remove console.log

**Review by nakiwuge — CHANGES_REQUESTED** [review]
@Jaman-dedy nice job, however, everything to do with actions and reducers should not be in this pull request because that is redux please remove them.
 Also, your commit message has no body please add a body and attend to the other comments I have left on this PR

File: src/serviceWorker.js [hunk 5]
```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```
Comment (nakiwuge) [inline src/serviceWorker.js]: Remove this file as we won't need it for this project. Read when serviceWorker is used  for your own information

**Review by nakiwuge — APPROVED** [review]

**Pull request merged**